src/test/scala/uk/gov/nationalarchives/draftmetadatavalidator/LambdaSpec.scala

```diff
@@ -24,41 +24,103 @@ class LambdaSpec extends ExternalServicesSpec {
     )
   }
 
-  def mockS3PutResponse(): StubMapping = {
+  def mockS3ErrorFilePutResponse(): StubMapping = {
     wiremockS3.stubFor(
-      put(urlEqualTo(s"/$consignmentId/sample.csv"))
+      put(urlEqualTo(s"/$consignmentId/draft-metadata-errors.json"))
         .willReturn(aResponse().withStatus(200))
     )
   }
 
-  "handleRequest" should "download the draft metadata csv file, validate and save to db if it has no errors" in {
+  "handleRequest" should "download the draft metadata csv file, validate, save error file to s3 and save to metadata to db if it has no errors" in {
```

Comment:
I would reword this slightly to: `... validate, save empty error file to s3 and ...`